trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    name = @nameable.correct_name
+    name.length > 10 ? name[0..10] : name
```

Comment:
- [ ] Your trimmer returns 11 characters, instead of 10 characters. Kindly change `[1..10]` to `[0..9]`.

  ```
  name.length > 10 ? name[0..9] : name
  ```

  ![image](https://github.com/yusufyusie/School-Library/assets/36542986/6c2e6a8c-403b-445b-a869-d6f611aabe75)